### ISSUE TITLE
Only link to guidance articles on UUTilsynet in our accessibility tab

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#5079]
When linking to WCAG Success Criterions I noticed that it was inconsistent where we linked to. Sometimes it was ["WCAG-standarden"](https://www.uutilsynet.no/wcag-standarden/111-ikke-tekstlig-innhold-niva/87) and other times it was ["veiledning"](https://www.uutilsynet.no/veiledning/111-ikke-tekstlig-innhold/1231). We should be consistent on what page we link to. I suggest that we link to "veiledning". 

I noticed it in our Accessiblity tabs on our component pages. [Breadcrumbs](https://designsystemet.no/no/components/docs/breadcrumbs/accessibility) have links to both types of pages. Maybe check the rest of our documentation as well.
